src/components/dialogs/renew.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <v-dialog min-width="32rem" max-width="45rem" :value="isVisible" persistent>
+    <v-card>
+      <v-tabs id="renew-tabs">
+        <v-tabs-items v-model="currentTab">
+
+          <v-tab-item>
+            <v-card-title class="d-flex justify-space-between">
+              <div>Staff Payment for Request Renewal</div>
+              <v-btn icon large class="dialog-close float-right" @click="hideModal()">
+                <v-icon>mdi-close</v-icon>
+              </v-btn>
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              <StaffPayment
+                ref="staffPaymentComponent"
+                @isValid="isStaffPaymentValid = $event"
+              />
+            </v-card-text>
+
+            <v-card-actions class="justify-center pt-6">
+              <v-btn
+                @click="goBack()"
+                id="renew-back-btn"
+                class="button-blue px-10"
+                :disabled="isLoadingPayment">Back</v-btn>
+              <v-btn
+                @click="confirmPayment()"
+                id="renew-submit-button"
+                class="primary px-5"
+                :loading="isLoadingPayment">Submit Renewal</v-btn>
+            </v-card-actions>
+          </v-tab-item>
+
+          <v-tab-item>
+            <v-card-title class="d-flex justify-space-between">
+              <div>Renew Name Request</div>
+              <v-btn icon large class="dialog-close float-right pa-0" @click="hideModal()">
+                <v-icon>mdi-close</v-icon>
+              </v-btn>
+            </v-card-title>
+
+            <v-card-text class="copy-normal">
+              <p v-if="!isRoleStaff" class="mb-8">
```

Comment:
Haha, yes, I'm going against my best practise of adding top whitespace whenever possible.

I believe this is an exception because it's a conditional element -- if it displays then we need whitespace before the next element, otherwise we don't.

Tell me if you feel strongly about this. I could remove the bottom margin here and add this to the next element:
```
   :class="{ 'mt-8': isRoleStaff }"
```

Reply:
... but then why pollute the next element with this element's state? 🤷‍♂️

Reply:
@andrepestana-aot Although I have now merged this PR, if you feel strongly that the next element should specify top margin instead of the existing code, let me know and I can submit another PR with that change.